cypress/support/step_definitions/dynamicCalls.js

```diff
@@ -348,6 +348,20 @@ Given(
  * Given 'third party app is launched' on 'auth' page
  */
 Given(/'(.+)' on '(.+)' page/, (validationObjectKey, page) => {
+  const requestMap = {
+    method: CONSTANTS.REQUEST_OVERRIDE_CALLS.GETAPPSTATE,
+    params: Cypress.env(CONSTANTS.CURRENT_APP_ID),
+  };
+
```

Comment:
[State validation successful: Current state is 'FOREGROUND' as expected]

this log change to 

fireLog.info(`State validation successful: Current state of '${appId}' app is '${response}' as expected`);

it will look something like 
State validation successful: Current state of apple_inc_apple_tv app is FOREGROUND as expected`